src/vs/workbench/services/workingCopy/common/untitledFileWorkingCopyManager.ts

```diff
@@ -167,6 +169,7 @@ export class UntitledFileWorkingCopyManager<M extends IUntitledFileWorkingCopyMo
 		// Handle untitled resource
 		else if (options.untitledResource?.scheme === Schemas.untitled) {
 			massagedOptions.untitledResource = options.untitledResource;
+			massagedOptions.scratchPad = options.scratchPad;
```

Comment:
I wonder: why not allow scratchpads even with associated resource? You could then save a scratchpad without asking the user for a path?

Reply:
they would just be titled `untitled-n` in that case, so we would probably need a way to differentiate them in the UI from non-scratchpad untitled files. It feels simplest to just leave that out for now.

Reply:
👍